pkg/components/contour/component.go

```diff
@@ -19,9 +19,10 @@ import (
 
 	"github.com/hashicorp/hcl/v2"
 	"github.com/hashicorp/hcl/v2/gohcl"
-	"github.com/kinvolk/lokomotive/pkg/components/util"
 
+	internaltemplate "github.com/kinvolk/lokomotive/internal/template"
```

Comment:
Q: Why is it imported as `internaltemplate`? Name conflicts?

Reply:
Can we import it as `internaltemplate` everywhere else? So it becomes easier to read code?

Reply:
yes Suraj, name conflicts with Go library `template` package.